src/config/Icon.tsx

```diff
@@ -58,6 +58,7 @@ export type IconProps = {
 };
 
 export const IconSet = Ionicons;
+export const IconSetName = 'ionicon';
```

Comment:
can be removed now i guess

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/commit/8288e099f4fea42ad97e3eec77138c2b84977454